tests-unified/GeniusYield/Test/Unified/OnChain/BetRef/Compiled.hs

```diff
@@ -0,0 +1,26 @@
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:target-version=1.0.0 #-}
+-- {-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:remove-trace #-}
+
+{-# LANGUAGE DataKinds         #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+
+module GeniusYield.Test.Unified.OnChain.BetRef.Compiled
+    ( betRefValidator
+    , BetRefParams (..)
+    , OracleAnswerDatum (..)
+    , BetRefDatum (..)
+    , BetRefAction (..)
+    ) where
+
+import qualified PlutusTx
+import           PlutusCore.Version                      (plcVersion100)
+
+
+import           GeniusYield.Test.Unified.OnChain.BetRef
+
+-- Since makeLift doesn't seem to work on BetRefParams. We just convert it to data and apply that instead.
```

Comment:
Don't know why this is the case, perhaps it would be fixed when we bump the plutus dependencies.